crud/common/schema.lua

```diff
@@ -197,8 +197,8 @@ function schema.truncate_row_trailing_fields(tuple, field_names)
     return tuple
 end
 
-function schema.wrap_func_result(space, func, args, opts)
-    dev_checks('table', 'function', 'table', 'table')
+function schema.wrap_func_result(box_obj, space_name, func, args, opts)
```

Comment:
Why do you need this change. I don't understand it if you really need it let's move it to separate patch with a good comment.

Reply:
I need `box_obj` to refetch sharding keys on schema reload.

Reply:
I'd call it "connection" (conn) because "box" usually about local box and it could confuse.

Reply:
renamed `box_obj` to `conn`